torch/nn/_intrinsic/quantized/modules/conv_relu.py

```diff
@@ -33,3 +48,11 @@ def forward(self, input):
                                                         self.dilation, self.groups,
                                                         float(self.scale), int(self.zero_point))
         return output.permute([0, 3, 1, 2])
+
+    @classmethod
+    def from_float(cls, mod):
+        if type(mod) == nniqat.ConvBnReLU2d:
+            mod.weight, mod.bias = \
+                fuse_conv_bn_weights(mod.weight, mod.bias, mod.running_mean,
+                                     mod.running_var, mod.eps, mod.gamma, mod.beta)
+        return super(ConvReLU2d, cls).from_float(mod)
```

Comment:
Wouldn't super return a nn.quantized.conv2d module instead of a nn._intrinsic.quantized.conv2drelu?

Reply:
no, we are using cls in the from_float

Reply:
I see, cls is pointing to conv2drelu